pages/exchange.tsx

```diff
@@ -32,7 +32,7 @@ const Exchange: ExchangeComponent = () => {
 						? t('exchange.page-title-currency-pair', {
 								baseCurrencyKey,
 								quoteCurrencyKey,
-								rate: formatCurrency(quoteCurrencyKey, inverseRate, {
+								rate: formatCurrency(quoteCurrencyKey, rate, {
```

Comment:
As you may notice, it used the `inverseRate` on page title. It explains why the order of the function signature is a bit strange.

Reply:
Now revert to inverted rate